src/test/system_tests/ceph_s3_tests/test_ceph_nsfs_s3_config_and_run_s3_tests.sh

```diff
@@ -0,0 +1,17 @@
+# This script objective is to run Ceph s3 tests
+# We use it inside of K8S job, see noobaa-tests-s3-job
```

Comment:
You don't use k8s job anymore, right? You can remove this comment.

Reply:
sure